scripts/configure_pyproject_toml.py

```diff
@@ -51,7 +65,16 @@ def parse_args():
     parsed_args = parse_args()
     pyproject_path = os.path.join(parsed_args.path, "pyproject.toml")
 
-    pyproject = toml.load(pyproject_path)
+    if not has_toml:
+        raise ImportError(
+            "A TOML parser is required to configure 'pyproject.toml'. "
+            "We support any of the following TOML parsers: [tomlkit, tomllib] "
+            "You can install tomlkit via `pip install tomlkit`, "
+            "or use Python 3.11 or above which natively offers the tomllib library."
+        )
```

Comment:
```suggestion
        raise ImportError(
            "A TOML parser is required to configure 'pyproject.toml'. "
            "We support any of the following TOML parsers: " + str(toml_libs) +
            "You can install tomlkit via `pip install tomlkit`, "
            "or use Python 3.11 or above which natively offers the tomllib library."
        )
```
It would be nice for maintenance to have this or any variant.